build.gradle

```diff
@@ -96,7 +96,7 @@ ext {
 }
 
 group = "mesosphere"
-version = "0.4.31-SNAPSHOT"
+version = "0.4.32-SNAPSHOT"
```

Comment:
This isn't needed for the go stuff -- go cli builds will always just pull latest from master branch